pandas/tests/test_graphics.py

```diff
@@ -3564,6 +3570,81 @@ def test_plotting_with_float_index_works(self):
         df.groupby('def')['val'].apply(lambda x: x.plot())
         tm.close()
 
+    def test_sharex_and_ax(self):
```

Comment:
You put this under TestDataFrameGroupByPlots btw. Maybe you meant TestDataFramePlots?